src/target/riscv/riscv.c

```diff
@@ -5847,6 +5848,52 @@ bool reg_is_initialized(const struct reg *reg)
 	return true;
 }
 
+static struct target *reg_get_target(const struct reg *reg)
```

Comment:
For clarity, I'd recommend to rename this function to `get_target_from_reg()`

```suggestion
static struct target *get_target_from_reg(const struct reg *reg)
```

Reply:
Addressed.